src/components/CoinListHeader/index.js

```diff
@@ -1,25 +1,129 @@
-import { Row } from "antd";
-import { ColumnHeader } from "./CoinHeader.styles";
+import React from "react";
+import { Row, Col } from "antd";
+import { FaCaretUp, FaCaretDown } from "react-icons/fa";
+import { RiHeartFill } from "react-icons/ri";
+import { ColumnHeader, FavIcon } from "./CoinHeader.styles";
 
-export const CoinListHeader = (props) => {
-  return (
-    <Row>
-      <ColumnHeader span={1}>
-        <div>Fav</div>
-      </ColumnHeader>
-      <ColumnHeader span={1}>
-        <div>Rank</div>
-      </ColumnHeader>
+export default class CoinListHeader extends React.Component {
+  state = {};
+  render() {
+    const { order, handleSort, sortBy } = this.props;
+    return (
+      <Row>
+        <ColumnHeader lg={{ span: 1 }}>
+          <FavIcon>
+            <RiHeartFill size="1rem" color="#ff7b7b" />
+          </FavIcon>
+        </ColumnHeader>
+        <ColumnHeader
+          lg={{ span: 1 }}
+          onClick={() => handleSort("market_cap_rank")}
+        >
+          <div>
+            #
+            {sortBy !== "market_cap_rank" ? (
+              ""
+            ) : order ? (
+              <FaCaretUp />
+            ) : (
+              <FaCaretDown />
+            )}
+          </div>
+        </ColumnHeader>
+        <ColumnHeader lg={{ span: 4 }} onClick={() => handleSort("name")}>
+          <div>
+            Name
+            {sortBy !== "name" ? "" : order ? <FaCaretUp /> : <FaCaretDown />}
```

Comment:
I'm not sure if chaining ternary expressions together is a good idea... performance issues? (not sure!)

Reply:
I will definitely try this Anton, thx, but will try at a later time